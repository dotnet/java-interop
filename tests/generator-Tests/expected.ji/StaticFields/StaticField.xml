<?xml version="1.0" encoding="UTF-8" ?>
<api>
	<package name="java.lang">
	    <class abstract="false" deprecated="not deprecated" final="false" name="Object" static="false" visibility="public">
	    </class>
	  </package>
	<package name="xamarin.test">
	  <class abstract="false" deprecated="not deprecated" extends="java.lang.Object" extends-generic-aware="java.lang.Object" 
	  		final="false" name="SomeObject" static="false" visibility="public">
	  		<field deprecated="not deprecated" final="true" name="Value" static="true" transient="false" type="int" type-generic-aware="int" visibility="public" volatile="false">
	        </field>
	        <field deprecated="not deprecated" final="false" name="Value2" static="true" transient="false" type="int" type-generic-aware="int" visibility="public" volatile="false">
	        </field>
	    <field deprecated="not deprecated" final="false" name="BooleanValue" static="true" transient="false" type="boolean" type-generic-aware="int" visibility="public" volatile="false" />
	    <field deprecated="not deprecated" final="false" name="CharValue" static="true" transient="false" type="char" type-generic-aware="int" visibility="public" volatile="false" />
	    </class>
	</package>
</api>

