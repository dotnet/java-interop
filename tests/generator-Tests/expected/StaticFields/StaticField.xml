<?xml version="1.0" encoding="UTF-8" ?>
<api>
	<package name="java.lang">
	    <class abstract="false" deprecated="not deprecated" final="false" name="Object" static="false" visibility="public">
	    </class>
	  </package>
	<package name="xamarin.test">
	  <class abstract="false" deprecated="not deprecated" extends="java.lang.Object" extends-generic-aware="java.lang.Object" 
	  		final="false" name="SomeObject" static="false" visibility="public">
	  		<field deprecated="not deprecated" final="true" name="Value" static="true" transient="false" type="int" type-generic-aware="int" visibility="public" volatile="false">
	        </field>
	        <field deprecated="not deprecated" final="false" name="Value2" static="true" transient="false" type="int" type-generic-aware="int" visibility="public" volatile="false">
	        </field>
	    </class>
	</package>
</api>

